www_src/components/option-panel/option-panel.jsx

```diff
@@ -0,0 +1,82 @@
+var React = require('react/addons');
+
+// Internal -- just the container
+var OptionPanel = React.createClass({
+  render: function () {
+    return (<div className="option-panel">
+      {this.props.children}
+    </div>);
+  }
+});
+
+var Checkbox = React.createClass({
+  mixins: [React.addons.LinkedStateMixin],
+  getDefaultProps: function () {
+    return {
+      linkState: this.linkState,
+      truthy: true,
+      falsey: false
+    };
+  },
+  onChange: function (e) {
+    var val = e.target.checked ? this.props.truthy : this.props.falsey;
+    this.valueLink.requestChange(val);
+  },
+  isTruthy: function () {
+    return this.valueLink.value === this.props.truthy;
```

Comment:
given that this isn't true or false, but truthy and falsey, should this be `==` rather than `===`?